Gemfile

```diff
@@ -30,6 +30,9 @@ group :development, :test do
   gem 'web-console', '~> 2.0'
   gem 'spring'
   gem 'minitest'
+  gem "minitest-rails", "~> 2.2.0"
```

Comment:
@anuj-verma I think we dont need `minitest-rails` gem for test env as we are rails testing framework. We have to include `minitest-rails` only if you want to use `spec` DSL. 

@jiren Am I correct?